### PR TITLE
A "happy path" that works for twitter oAuth and weVote login on iOS

### PR DESCRIPTION
**Pull request by SailingSteve**
An intermediate checkin.
Lots of commented out code, that I will address.
This should get us into shape so multiple people can work on iOS at once.
Next step is merging this from fluxbranch to develop.

### CODE REVIEW

File: src/js/scenes/OldMain/Root.js [hunk 1]
```diff
@@ -17,26 +17,26 @@ We are no longer using "react-router-native" in this way
 
 
 /****************************** ROUTE-COMPONENTS ******************************/
-//import About from "./routes/More/About";
+//import About from "./routes/SignIn/About";
```
Comment (DaleMcGrew) [inline src/js/scenes/OldMain/Root.js]: Hi Steve, it looks like there was a search-and-replace from "More" to "SignIn" that wasn't what you had planned.

File: src/js/config.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
 // Note that we import these values into "web_app_config" (so we can search for it)
```
Comment (DaleMcGrew) [inline src/js/config.js]: Hi Steve, please remove config.js from this Pull Request.

File: src/js/components/Widgets/ReadMore.js [hunk 3]
```diff
@@ -41,7 +41,7 @@ export default class ReadMore extends Component {
           num_of_lines = 3;
         }
         if (link_text === undefined) {
-          link_text = "More";
+          link_text = "SignIn";
```
Comment (DaleMcGrew) [inline src/js/components/Widgets/ReadMore.js]: Hi Steve, it looks like there was a search-and-replace from "More" to "SignIn" that wasn't what you had planned.

**Comment by DaleMcGrew** [comment]
Hi @SailingSteve, thank you for all of this great work. I added some comments for you to resolve before I can approve this pull request.

File: src/js/scenes/OldMain/Root.js (hunk 1 above)
Comment (SailingSteve) [reply src/js/scenes/OldMain/Root.js]: Must have been the Webstorm "refactor"  ... good catch.

**Comment by DaleMcGrew** [comment]
👍 Thank you!

**Comment by SailingSteve** [comment]
Issue #51

**Pull request merged**